flexget/plugins/sites/descargas2020.py

```diff
@@ -73,13 +66,22 @@ def parse_download_page(self, url):
             if match:
                 torrent_id = match.group(1)
         if not torrent_id:
-            torrent_id_prog = re.compile('function openTorrent.*\n.*\{.*(\n.*)+window\.location\.href =\s*\".*\/(\d+).*\";')
-            torrent_ids = soup.findAll(text=torrent_id_prog)
             log.debug('torrent ID not found, searching openTorrent script')
+            if 'descargas2020.com' in url:
+                torrent_id_prog = re.compile('function openTorrent.*\n.*\{.*(\n.*)+window\.location\.href =\s*\"(.*\/\d+_-.*[^\/])\/?\";')
+            else:
+                torrent_id_prog = re.compile('function openTorrent.*\n.*\{.*(\n.*)+window\.location\.href =\s*\"(.*\/\d+_.*)\";')
+            torrent_ids = soup.findAll(text=torrent_id_prog)
             if torrent_ids:
                 match = torrent_id_prog.search(torrent_ids[0])
                 if match:
                     torrent_id = match.group(2)
+                    if 'descargas2020.com' in url:
+                        return (torrent_id.replace('descargar-torrent','download') + '.torrent')
+                    else:
+                        response = task.requests.get(torrent_id)
+                        response = task.requests.get(response.url)
```

Comment:
I don't like the re-use of variable here. Makes it unclear what's going on. It also _needs_ a try-except.

Reply:
It's because there are redirects. but right now the feed does not work, maybe it's better to remove it for the moment. (It's the "else" for tvsinpagar, the feed is empty actualy)